src/message_filter.rs

```diff
@@ -74,6 +74,7 @@ impl<Message: Hash> MessageFilter<Message> {
     }
 
     /// Removes any expired messages, then returns whether `message` exists in the filter or not.
+    #[allow(unused)]
```

Comment:
I'm not against this (also a few in previous commit), as a transitional state within the PR; but at the end of this PR, can we do something such as
```
git diff origin/fleming | grep unused
```
And make sure to delete any dead code?